### ISSUE TITLE
Missing Data with Numerical Columns handled incorrectly

### ISSUE THREAD

**Issue by junder873** [Hasnep/DataSkimmer.jl#41]
When a column is a Union with type missing and a number, skim incorrectly categorizes it as a categorical column. For example, starting with the example provided (using the iris dataset):

```julia
allowmissing!(iris, :SepalLength)
skim(iris)
```

makes SepalLength go to categorical instead of numeric. It seems the package is designed to keep this as a numeric column since there is "Missings" there as well.

Output:
```
┌─────────────────────┬───────────┐
│                Type │ DataFrame │
│             N. rows │       150 │
│             N. cols │         5 │
│     N. numeric cols │         3 │
│ N. categorical cols │         2 │
│    N. datetime cols │         0 │
└─────────────────────┴───────────┘

3 numeric columns
┌─────────────┬─────────┬──────────┬──────────┬──────┬──────┬──────┬──────┬──────┬───────┐
│        Name │    Type │ Missings │ Complete │ Mean │ Std. │ Min. │ Med. │ Max. │ Hist. │
├─────────────┼─────────┼──────────┼──────────┼──────┼──────┼──────┼──────┼──────┼───────┤
│  SepalWidth │ Float64 │        0 │   100.0% │ 3.06 │ 0.44 │  2.0 │  3.0 │  4.4 │ ▁▃▄▂▁ │
│ PetalLength │ Float64 │        0 │   100.0% │ 3.76 │ 1.77 │  1.0 │ 4.35 │  6.9 │ ▃▁▂▃▁ │
│  PetalWidth │ Float64 │        0 │   100.0% │  1.2 │ 0.76 │  0.1 │  1.3 │  2.5 │ ▃▁▃▂▂ │
└─────────────┴─────────┴──────────┴──────────┴──────┴──────┴──────┴──────┴──────┴───────┘

2 categorical columns
┌─────────────┬─────────────────────────────────┬──────────┬──────────┐
│        Name │                            Type │ Missings │ Complete │
├─────────────┼─────────────────────────────────┼──────────┼──────────┤
│ SepalLength │         Union{Missing, Float64} │        0 │   100.0% │
│     Species │ CategoricalValue{String, UInt8} │        0 │   100.0% │
└─────────────┴─────────────────────────────────┴──────────┴──────────┘
```

**Comment by Hasnep** [comment]
Thank you for this report, I didn't see it until now for some reason. Should have this fixed by the end of the day. :crossed_fingers:

**Comment by Hasnep** [comment]
Should be fixed in version 0.2.1.

Example:

```
allowmissing!(iris, :SepalLength)
skim(iris)
```

```
┌─────────────────────┬───────────┐
│                Type │ DataFrame │
│             N. rows │       150 │
│             N. cols │         5 │
│     N. numeric cols │         4 │
│ N. categorical cols │         1 │
│    N. datetime cols │         0 │
└─────────────────────┴───────────┘

4 numeric columns
┌─────────────┬─────────────────────────┬──────────┬──────────┬──────┬──────┬──────┬──────┬──────┬───────┐
│        Name │                    Type │ Missings │ Complete │ Mean │ Std. │ Min. │ Med. │ Max. │ Hist. │
├─────────────┼─────────────────────────┼──────────┼──────────┼──────┼──────┼──────┼──────┼──────┼───────┤
│ SepalLength │ Union{Missing, Float64} │        0 │   100.0% │ 5.84 │ 0.83 │ 4.30 │ 5.80 │ 7.90 │ ▂▃▃▂▁ │
│  SepalWidth │                 Float64 │        0 │   100.0% │ 3.06 │ 0.44 │ 2.00 │ 3.00 │ 4.40 │ ▁▃▄▂▁ │
│ PetalLength │                 Float64 │        0 │   100.0% │ 3.76 │ 1.77 │ 1.00 │ 4.35 │ 6.90 │ ▃▁▂▃▁ │
│  PetalWidth │                 Float64 │        0 │   100.0% │ 1.20 │ 0.76 │ 0.10 │ 1.30 │ 2.50 │ ▃▁▃▂▂ │
└─────────────┴─────────────────────────┴──────────┴──────────┴──────┴──────┴──────┴──────┴──────┴───────┘

1 categorical column
┌─────────┬─────────────────────────────────┬──────────┬──────────┐
│    Name │                            Type │ Missings │ Complete │
├─────────┼─────────────────────────────────┼──────────┼──────────┤
│ Species │ CategoricalValue{String, UInt8} │        0 │   100.0% │
└─────────┴─────────────────────────────────┴──────────┴──────────┘
```
